src/win/process.c

```diff
@@ -886,17 +932,208 @@ void uv__process_endgame(uv_loop_t* loop, uv_process_t* handle) {
   uv__handle_close(handle);
 }
 
+int uv_pty_resize(uv_process_t* process,
+                  unsigned short cols,
+                  unsigned short rows) {
+  if (process->pty_handle == NULL)
+    return UV_EINVAL;
+  HANDLE hLibrary = LoadLibraryExW(L"kernel32.dll", 0, 0);
+  // Error loading kernel32.dll: (error code %i)
+  if (!hLibrary)
+    return uv_translate_sys_error(GetLastError());
+
+  PFNRESIZEPSEUDOCONSOLE pfnResize = (PFNRESIZEPSEUDOCONSOLE)GetProcAddress((HMODULE)hLibrary,"ResizePseudoConsole");
```

Comment:
It would be nice if these could get lazy initialized as global variables. `LoadLibrary` and `GetProcAddress` are both fairly expensive calls. An SRWLOCK may be suitable for this. I would also suggest passing `LOAD_LIBRARY_SEARCH_SYSTEM32`, but for `kernel32.dll` in particular I'd just use `GetModuleHandleW`. It won't be getting unloaded.